torch/onnx/_internal/_exporter_legacy.py

```diff
@@ -419,10 +419,15 @@ def enable_fake_mode():
         ...     arg1,
         ...     export_options=export_options
         ... )
+        ... onnx_program.apply_weights(MyModel().state_dict())
         >>> # Saving model WITHOUT initializers
-        >>> onnx_program.save("my_model_without_initializers.onnx")
+        >>> onnx_program.save(
+        ...     "my_model_without_initializers.onnx",
+        ...     include_initializers=False,
+        ...     keep_initializers_as_inputs=True,
+        ... )
         >>> # Saving model WITH initializers
-        >>> onnx_program.save("my_model_with_initializers.onnx", model_state=MyModel().state_dict())
+        >>> onnx_program.save("my_model_with_initializers.onnx")
```

Comment:
Would it be possible to further change the example such that:

1. `torch.onnx.dynamo_export` becomes the `dynamo=True` usage
2. export_options is dropped

This should make the test pass. Thanks!